src/test/ui/imports/empty-extern-arg-wasm.stderr

```diff
@@ -0,0 +1,9 @@
+error: extern location for std does not exist: 
+
+error[E0463]: can't find crate for `std`
+   |
+   = note: the `wasm32-unknown-unknown` target may not be installed
```

Comment:
You could also use normalization to get rid of these two lines. Then you don't need to duplicate tests.

Reply:
I have e.g. sued the following to normalize away entire lines in other tests:
```
// normalize-stderr-test "thread.*panicked.*Metadata module not compiled.*\n" -> ""
// normalize-stderr-test "note:.*RUST_BACKTRACE=1.*\n" -> ""
```

Reply:
Hm, this is great news.
For some reason I thought the normalization works per-line.

Reply:
Much simpler to fix platform-dependent tests this way.

Reply:
I think the matching of the normalization pattern *is* per-line -- but if you include the final `\n`, you can remove the entire line. And then that's handled correctly when comparing later.

Reply:
I'll have a look at this tomorrow; in the meantime feel free to r- or we can do this in a follow up PR if that's alright with y'all.